### PR TITLE
Use new icon for Google Lens

### LINKED ISSUE #3659: [FEATURE] Change lens icon to new icon
### Describe the feature

Change the old lens icon to new google lens icon on qsb

### Why would it be useful to add?

To make it more identical to pixel launcher 

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

2048px-Google_Lens_Icon svg
This logo is the new lens icon. 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by validcube**
## Description
Use the new Google Lens icon that was introduced in Google I/O 2023 event.

Fix #3659

### Testing

🚧 (recently out of) Draft, changes haven't been tested *yet*!

## Type of change

✅ General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lawnchair/res/drawable/ic_lens_color.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
     xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:id="@id/qsbIconTintPrimary" android:drawable="@drawable/ic_lens_color_1" />
     <item android:id="@id/qsbIconTintQuaternary" android:drawable="@drawable/ic_lens_color_2" />
-    <item android:id="@id/qsbIconTintPrimary" android:drawable="@drawable/ic_lens_color_3" />
     <item android:id="@id/qsbIconTintTertiary" android:drawable="@drawable/ic_lens_color_4" />
+    <item android:id="@id/qsbIconTintPrimary" android:drawable="@drawable/ic_lens_color_3" />
```
Comment (validcube) [inline lawnchair/res/drawable/ic_lens_color.xml]: Do I attempt to fix svg clipping or leave it like that... it looks kinda ugly to me.

Comment (KTibow) [reply lawnchair/res/drawable/ic_lens_color.xml]: send a screenshot of what you're talking about

File: lawnchair/res/drawable/ic_lens.xml
Comment (KTibow) [inline lawnchair/res/drawable/ic_lens.xml:1]: where did this icon come from?

Comment (validcube) [reply lawnchair/res/drawable/ic_lens.xml:1]: The icon come from 

https://commons.wikimedia.org/wiki/File:Google_Lens_Icon.svg

File: lawnchair/res/drawable/ic_lens_color.xml (hunk 1 above)
Comment (validcube) [reply lawnchair/res/drawable/ic_lens_color.xml]: If I arrange the drawable assets in numerical order (1 to 5)

`ic_lens_color_4` will display on top of `ic_lens_color_3`, to be honest I can just rename the file...

image

**Review by Goooler — APPROVED** [review]

**Pull request merged**